packages/components/src/clipboard-button/README.md

```diff
@@ -24,3 +28,39 @@ const MyClipboardButton = () => {
 	);
 };
 ```
+
+## Props
+
+The component accepts the following props:
+
+### className
+
+The class that will be added to the classes of the underlying `<Button>` component.
+
+- Type: `string`
+- Required: no
+
+### text
+
+The text that will be copied to the clipboard.
+
+- Type: `string`
+- Required: no
```

Comment:
From a quick look at the code, it seems like `text` would be required — WDYT?

Reply:
This totally makes sense to be marked as required. I checked this and it doesn't fire errors but I think the idea of it being required to use the component rather than required to not break it is a good distinction.